test/test_images/failing/failing.go

```diff
@@ -25,8 +25,7 @@ import (
 func main() {
 	log.Println("Started...")
 
-	// Sleep for 10 seconds to force a race condition, where this
-	// container becomes ready if no readinessProbe is set.
+	// Sleep for 10 seconds and then crash.
```

Comment:
Why drop this? This test used to break often without the additional sleep. :thinking:

Reply:
I don't think I dropped the sleep though :)